frontend/src/lib/constants.tsx

```diff
@@ -215,7 +215,6 @@ export const FEATURE_FLAGS = {
     INSIGHT_VARIABLES: 'insight_variables', // owner: @Gilbert09 #team-data-warehouse
     WEB_EXPERIMENTS: 'web-experiments', // owner: @team-feature-success
     BIGQUERY_DWH: 'bigquery-dwh', // owner: @Gilbert09 #team-data-warehouse
-    REPLAY_DEFAULT_SORT_ORDER_EXPERIMENT: 'replay-order-by-experiment', // owner: #team-replay
```

Comment:
@richard-better i removed the experiment rather than maintain it through the refactor - i think the feedback on my idea showed it was confusing enough that we should try a different tack